### DISCUSSION TITLE
placement option is wrong

### DISCUSSION THREAD

**Discussion by snmatsui** [Akryum/v-tooltip#731] [Q&A]
position modifier and placement option are wrong.
A "left" tooltip appears on the right.

<img width="774" alt="2021-05-28_21h14_30" src="https://user-images.githubusercontent.com/10430686/119982383-baff7d80-bff9-11eb-957f-bc3177cc8f6c.png">

**Comment by Akryum** [comment]
By default, if there is not enough space on the left it will flip to the right.

**Reply by snmatsui** [reply]
I understood that is PopperV1's feature. Thank you.
